core/comet/service.go

```diff
@@ -0,0 +1,63 @@
+package comet
+
+import (
+	"context"
+	"time"
+)
+
+// Service is an interface that can be used to get information specific to Comet
```

Comment:
s/Service/InfoService/

Punctuate your godoc so it reads properly when rendered on pkg.go.dev.